addOns/automation/src/main/java/org/zaproxy/addon/automation/ContextWrapper.java

```diff
@@ -90,7 +91,13 @@ public ContextWrapper(Context context) {
         // Contexts dont actually define the starting URL, but we need at least one
         for (String url : context.getIncludeInContextRegexs()) {
             if (url.endsWith(".*")) {
-                this.addUrl(url.substring(0, url.length() - 2));
+                String urlStr = env.replaceVars(url.substring(0, url.length() - 2));
```

Comment:
Should we trim before the endsWith just to be sure there's no white space?

Reply:
There will not be any spaces unless they use quotes and then include the quote at the end.
TBH the main use case for this is for contexts generated by the AF, and that will not use quotes.